docs/manual/java/gettingstarted/index.toc

```diff
@@ -2,10 +2,8 @@ IntroGetStarted:Introduction
 JavaPrereqs:Lagom Java prerequisites;next=GettingStartedMaven,GettingStartedSbt
 GettingStartedMaven:Creating and running Hello World with Maven;next=UnderstandHello
 GettingStartedSbt:Creating and running Hello World with sbt;next=UnderstandHello
-UnderstandHello:Understanding Hello World;next=MavenIDEs,EclipseSbt,IntellijSbt
-MavenIDEs:Integrating Maven projects with an IDE;next=LagomExamples
-EclipseSbt:Integrating sbt with Eclipse;next=LagomExamples
-IntellijSbt:Integrating sbt with IntelliJ;next=LagomExamples
+UnderstandHello:Understanding Hello World
 LagomExamples:Lagom examples
+WhatsNext:What's Next?
```

Comment:
More on the subject https://medium.com/@jsaito/making-a-case-for-letter-case-19d09f653c98